### PR TITLE
WINDUPRULE-1049 fix broken links

### PR DESCRIPTION
**Pull request by m-brophy**
https://issues.redhat.com/browse/WINDUPRULE-1049

### CODE REVIEW

File: rules/rules-reviewed/eap7/weblogic/weblogic-webservices.windup.xml [hunk 1]
```diff
@@ -152,7 +152,7 @@
                     </message>
                     <link href="https://docs.oracle.com/cd/E13222_01/wls/docs90/webserv/anttasks.html#1039270" title="Oracle ClientGen Ant Task Reference"/>
                     <link href="https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.0/html-single/developing_web_services_applications/#using_jax_ws_tools" title="Developing Web Services Applications - Using JAX-WS Tools"/>
-                    <link href="https://jax-ws.java.net/nonav/2.2.6/docs/ch04.html#tools-wsgen" title="JAX-WS - wsgen"/>
+                    <link href="https://www.ibm.com/docs/en/was/9.0.5?topic=applications-wsgen-command-jax-ws" title="JAX-WS - wsgen"/>
```
Comment (mrizzi) [inline rules/rules-reviewed/eap7/weblogic/weblogic-webservices.windup.xml:155]:
```suggestion
                    <link href="https://docs.oracle.com/javase/8/docs/technotes/tools/unix/wsgen.html" title="JAX-WS - wsgen"/>
```

**Review by mrizzi — COMMENTED** [review]
Just a suggestion for a link towards the "official" Java `wsgen` tool

**Comment by m-brophy** [comment]
> Just a suggestion for a link towards the "official" Java `wsgen` tool

Thanks Marco

**Review by mrizzi — APPROVED** [review]
LGTM, thanks @m-brophy

**Pull request merged**